schemas/src/shared/Refresh.json

```diff
@@ -7,6 +7,10 @@
 			"type": "Action.Execute",
 			"description": "The action to be executed to refresh the card. Clients can run this refresh action automatically or can provide an affordance for users to trigger it manually."
 		},
+		"expires": {
+			"type": "string",
+			"description": "Hint to host that a refresh to card wouldn't be needed till timestamp. The timestamp is expected to be in ISO-8601 Instant format.  E.g. 2022-01-01T12:00:00Z"
+		},
```

Comment:
Suggested description: "A timestamp that informs a Host when the card content has expired, and that it should trigger a refresh as appropriate. The format is ISO-8601 Instant format. E.g., 2022-01-01T12:00:00Z"

Reply:
Updated as suggested